tests/lib/rules/no-unused-vars.js

```diff
@@ -141,6 +141,12 @@ ruleTester.run("no-unused-vars", rule, {
             output: null,
             errors: [{ type: "Identifier" }]
         },
+        {
+            code: "var [a, b] = c",
+            output: null,
+            parserOptions: { ecmaVersion: 6 },
+            errors: [{ type: "Identifier" }, { type: "Identifier" }]
+        },
```

Comment:
seems it can also be fixed. an enhencement. :(

Reply:
Array destructing is order-sensitive.

Reply:
that's to say, the fixer should not remove the commas.

Reply:
Done.